### PR TITLE
ITSASU-274 - Migrated accounting period check page

### PR DESCRIPTION
**Pull request by AlexRimmerHMRC**

### CODE REVIEW

File: it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/CheckAccountingPeriodControllerISpec.scala [hunk 1]
```diff
@@ -91,7 +98,6 @@ class CheckAccountingPeriodControllerISpec extends ComponentSpecBase with ViewSp
     "return SEE_OTHER when selecting Yes and send an Audit" in new PostSetup(Some(Yes)) {
       val expectedAuditContainsYes: JsValue = Json.parse(
         """{ "userType" : "individual", "eligible" : "true" , "answer" : "yes", "question": "standardAccountingPeriod" }""")
-      verifyAudit()
```
Comment (llmikeyj) [inline it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/CheckAccountingPeriodControllerISpec.scala:94]: Is it that there is nothing to verify, or that this is being done elsewhere (different concern)?

**Review by llmikeyj — APPROVED** [review]
LGTM, minor query about the audit verify

File: it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/CheckAccountingPeriodControllerISpec.scala (hunk 1 above)
Comment (AlexRimmerHMRC) [reply it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/CheckAccountingPeriodControllerISpec.scala:94]: The line underneath does a verifyAuditContains, which already verifies the audit, but with more detail, so we don't need the verifyAudit() specifically

Comment (llmikeyj) [reply it/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/controllers/principal/CheckAccountingPeriodControllerISpec.scala:94]: ok cool

**Pull request merged**